torch/_torch_docs.py

```diff
@@ -4709,6 +4709,45 @@ def merge_dicts(*dicts):
     tensor([  2.,   4.,   8.,  16.])
 """.format(**common_args))
 
+add_docstr(torch.nanprod,
+           r"""
+nanprod(input, dtype=None) -> Tensor
+
+Returns the product of all elements excluding ``NaN`` values in the
```

Comment:
Returns the product of all elements in the input tensor, treating "NaN" values as ones.

Reply:
Done, thanks!